### ISSUE TITLE
Gravity Forms required text fails contrast in dark mode

### ISSUE THREAD

**Issue by amberhinds** [wpa11yday/wpaccessibilityday#176]
Example: https://wpaccessibility.day/?page_id=6128&preview=1&_ppp=03f30360bb

<img width="1865" height="677" alt="Image" src="https://github.com/user-attachments/assets/7424249c-8c1f-45c3-aa3f-8535c7adcbf5" />

This color makes the required indicator the same color as the background so it is invisible:

```
#page #gform_wrapper_1 .gfield_required, #page #gform_wrapper_24 .gfield_required {
    color: var(--bg);
}
```

It should be white or light pink or something.
